src/SourceBuild/content/test/Microsoft.DotNet.SourceBuild.SmokeTests/SdkContentTests.cs

```diff
@@ -46,10 +47,36 @@ private void WriteTarballFileList(string? tarballPath, string outputFileName, bo
         fileListing = BaselineHelper.RemoveRids(fileListing, isPortable);
         fileListing = BaselineHelper.RemoveVersions(fileListing);
         IEnumerable<string> files = fileListing.Split(Environment.NewLine).OrderBy(path => path);
+        files = FilteredFileList(
+                    files,
+                    GetExclusionFilters(
+                        Path.Combine(BaselineHelper.GetAssetsDirectory(), "SdkDiffExclusions.txt"),
+                        tarballPath == Config.MsftSdkTarballPath ? "msft" : "sb"));
 
         File.WriteAllLines(outputFileName, files);
     }
 
+        private static IEnumerable<string> FilteredFileList(IEnumerable<string> files, IEnumerable<string> exclusions)
```

Comment:
To follow standard naming conventions, the method name should contain a verb.  Perhaps including the _Exclusion_ term would help the readability as it indicates what filter is being applied.

```suggestion
        private static IEnumerable<string> RemoveExclusions(IEnumerable<string> files, IEnumerable<string> exclusions)
```

Reply:
Sure, I'll update that. The idea was that the method returns a list that is filtered.

Reply:
Fixed with https://github.com/dotnet/installer/pull/15650/commits/9392ef335293dedd8c24c98ba29c908a27e944b5